frontend/src/App.js

```diff
@@ -7,7 +7,16 @@ import UserTokenTest from "./pages/UserTokenTest";
 import Layout from "./components/Layout";
 import RequireAuth from "./components/RequireAuth";
 import Profile from "./pages/Profile";
+import MatchTest from "./pages/MatchTest";
+import Collaboration from "./pages/Collaboration";
 import PersistLogin from './components/PersistLogin';
+
+// Text to simulate question. This should be provided 

```

Comment:
This might be better retrieved within the Collaboration element based on the URL parameters

Let's leave it as is for now, pending updates from question/matching services to finalize

Reply:
agreed, I'm only putting the placeholder text there to style the front end.